pkg/cmd/types.go

```diff
@@ -57,5 +68,39 @@ func (i *ImageInspectorOptions) Validate() error {
 	if len(i.Serve) == 0 && i.Chroot {
 		return fmt.Errorf("Change root can be used only when serving the image through webdav")
 	}
+	if len(i.ScanType) > 0 && len(i.ScanResultsDir) == 0 {
+		return fmt.Errorf("scan-result-dir must be specified with scan-type")
+	}
+	if len(i.ScanResultsDir) > 0 && len(i.ScanType) == 0 {
+		return fmt.Errorf("scan-result-dir can be used only when spacifing scan-type")
+	}
+	if len(i.ScanResultsDir) > 0 {
+		fi, err := os.Stat(i.ScanResultsDir)
+		if err != nil || os.IsNotExist(err) {
```

Comment:
@enoodle who's creating this directory in the pod use-case? Have you tested this with ManageIQ? Can you provide an example of the command line that will be used?